en/mkdocs.yml

```diff
@@ -295,11 +295,12 @@ nav:
           - Product REST APIs:
                 - RESTful APIs: develop/product-apis/restful-apis.md
                 - Getting Started:
-                    - Publisher API v1: develop/product-apis/getting-started/guide-publisher-v0.16.md 
+                    - Publisher API v1: develop/product-apis/getting-started/guide-publisher-v0.16.md
```

Comment:
we have two publisher API's of the same version. Is ths correct?

Reply:
https://github.com/wso2/docs-apim/pull/1527/commits/7eb51e9d35eabdd9d931a5fc427d90a1b519c011